var/spack/repos/builtin/packages/magma/package.py

```diff
@@ -14,9 +14,9 @@ class Magma(CMakePackage, CudaPackage):
     """
 
     homepage = "http://icl.cs.utk.edu/magma/"
-    url = "http://icl.cs.utk.edu/projectsfiles/magma/downloads/magma-2.2.0.tar.gz"
-    maintainers = ['luszczek']
```

Comment:
You accidentally deleted the maintainer

Reply:
Thanks. Actually it wasn't accidental - I wasn't sure of the purpose of the clause as many other packages don't have it, but anyway, I am thinking to maintain it from now on when needed.

Reply:
In that case, you can add your name to the list. `maintainers` is a list of GitHub accounts of people who have volunteered to maintain a Spack package. It gives us someone to ping when a user reports a build error, or when someone submits a PR to modify the package.